Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -29,6 +29,9 @@ class AbstractStagingTexture;
 class PointerWrap;
 struct VideoConfig;
 
+inline static const std::string efb_dump_prefix = "efb_frame";
```

Comment:
```suggestion
constexpr std::string_view efb_dump_prefix = "efb_frame";
```

for both of these so that they don't add static constructors